randovania/games/fusion/logic_database/Sector 5 ARC.txt

```diff
@@ -100,6 +100,9 @@ Extra - room_id: [3, 6]
   * Layers: default
   * L3 Hatch to Gerubus Gully/Door to Nightmare Training Grounds
   * Extra - door_idx: (7,)
+  > Door to Training Aerie
+      # https://youtu.be/Qe9eMPmoUcU
+      Level 0 Locks and Level 3 Locks and Speed Booster and Shinespark Tricks (Beginner) and Wall Jump (Advanced) and Disabled Door Lock Rando and Disabled Entrance Rando
```

Comment:
⛏️ Would argue "intermediate" on the shinespark difficulty only because of reset time
⛏️ Unsure if the L0 requirement is needed since all the doors on the shinespark path are L3s

Reply:
our definition for shinespark beginner is just a simple shinespark of the ground which this fits so I think that it's safe as is (unless the window to charge it is rather tight then happy to bump it up).
Yep I just misremembered the colors, will update.

Reply:
Nah, we're good on the difficulty then